VirtusLab-Open-Source/strapi-plugin-comments
# Feat/moderator responses in threads

## Ticket

[55](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/issues/55)

## Summary

What does this PR do/solve?

Allows moderator to respond directly in thread, start a new thread under comment and update comment. 

## Test Plan

1. Add some comments
2. Start new threads by submitting new comments under created ones
3. Update your submitted comment, should only work if you are the author
4. Delete your comment, should only work if you are the author
5. Block thread and re-open it by submitting new comment
